dao/src/main/java/com/epam/esm/dao/entity/Users.java

```diff
@@ -5,20 +5,27 @@
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.EntityListeners;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
 import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.OneToMany;
 import javax.persistence.Table;
 import java.math.BigDecimal;
-import java.util.HashSet;
+import java.util.Objects;
 import java.util.Set;
 
+import static com.epam.esm.dao.entity.Role.USER;
+
 @Entity
 @Table(name = "users")
 @EntityListeners(value = AuditListener.class)
-public class User implements TableEntity {
+public class Users implements TableEntity {
```

Comment:
Does it really necessary to rename entity to Users? It is very uncomfortable and unintuitive

Reply:
The decision of naming this entity was made in consultation with my mentor.